pkg/translator/loki/convert.go

```diff
@@ -170,14 +171,23 @@ func convertLogToLogfmtEntry(lr plog.LogRecord, res pcommon.Resource, scope pcom
 	}, nil
 }
 
+func convertLogToLogRawEntry(lr plog.LogRecord) (*push.Entry, error) {
+	return &push.Entry{
+		Timestamp: timestampFromLogRecord(lr),
+		Line:      lr.Body().Str(),
```

Comment:
The body could be `AnyValue`, whereas `lr.Body().Str()` will return an empty string if `Body().Type()` is not `ValueTypeStr`.
Here should be a smarter way to convert `lr.Body()` to the line, considering not only a string could be in `lr.Body()`

Reply:
Using [AsString()](https://github.com/open-telemetry/opentelemetry-collector/blob/3c791d86a28249b751f18827958f2534da625398/pdata/pcommon/value.go#L364) to convert body to string.